site/write_ups/2023/29-03-23_writeup.md

```diff
@@ -0,0 +1,127 @@
+
+
+*When do we decide when not to build something?*
+
+
+Considering the risk of unforeseen harms, at what point do you decide not to build technology? How far away from harm do you have to be down the causal chain to have responsibility for the consequences of your work? Is it when there are very clear implications that Huw’s work could be used for genocide, or something else awful, like a boring PhD thesis? How much harm will he cause his reviewers? Should he be stopped now?
+
+It is important to distinguish the point at which you halt development, from where mitigation techniques should be brought in. Maybe someone could develop some sort of signposting to help this... Something like [Data Hazards](https://datahazards.com/) labels? Just a thought.
+
+To understand where these thresholds are, the podcast talks about projects in the domain of high risk, high consequence. If there is a high probability that things could go wrong, and if things go wrong they go really wrong, maybe you should stop. [An article we read in a previous reading group](https://journals.sagepub.com/doi/10.1177/20539517221146122) provided a quite simple but useful heuristic for machine learning.  The paper suggests that ML should be used for well-defined, ‘learnable’, and low stakes problems, with relatively small ranges of subjectivity. This might help when we are trying to think about what could be done, or go wrong with, the models we build.
+
+
+*Removing responsibility through depersonalisation?*
+
+
+There is often the reasoning in tech that “we’re only making a tool” and thus aren’t accountable for misapplications of our work. However, it is difficult to know where this line should be drawn. The scope of some tools might be quite generic with wide ranging application domains, some of which could be harmful. For example, license plate recognition could be based off of generic text/image recognition. If you follow this reasoning too far, it can lead to unintuitive outcomes. Should we blame the inventors of the wheel for tanks? How many degrees of separation do you need to claim the technology that you developed is nothing to do with harm that is caused? This highlights the ambiguity of ethical reasoning.
+To define the tipping point of accountability, some of us had the intuition that if there’s a harm you could reasonably foresee then you hold some sort of responsibility to make efforts towards mitigation. However, if it is completely out of the realm of what you could conceive then you are off the hook. We also thought that it might depend on what you could do personally to mitigate the risk of bad things happening; what is in your personal control.
```

Comment:
New paragraph
```suggestion

To define the tipping point of accountability, some of us had the intuition that if there’s a harm you could reasonably foresee then you hold some sort of responsibility to make efforts towards mitigation. However, if it is completely out of the realm of what you could conceive then you are off the hook. We also thought that it might depend on what you could do personally to mitigate the risk of bad things happening; what is in your personal control.
```